app/src/main/java/org/simlar/utils/Util.java

```diff
@@ -139,4 +139,14 @@ private static SimpleDateFormat createSimpleDateFormat(final long milliSeconds)
 		}
 		return service;
 	}
+
+	public static boolean defaultTo(final Boolean value, final boolean defaultValue)
```

Comment:
I wouldn't convert `Boolean`s to `boolean`s this way. See [BooleanUtils](http://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/BooleanUtils) for  methods with readable names: `isFalse`, `isNotFalse`, `isNotTrue`, and `isTrue`.